### PR TITLE
Fix failing breadcrumb test 

### LINKED ISSUE #198: Test error - The text "Page 1" was not found anywhere in the text of the current page.
```
Testing Drupal\Tests\localgov_directories_page\Functional\PathIntegrationTest
Path Integration (Drupal\Tests\localgov_directories_page\Functional\PathIntegration)
 ✘ Directory integration
   ┐
   ├ Behat\Mink\Exception\ResponseTextException: The text "Page 1" was not found anywhere in the text of the current page.
   │
   ╵ /app/vendor/behat/mink/src/WebAssert.php:811
   ╵ /app/vendor/behat/mink/src/WebAssert.php:262
   ╵ /app/web/core/tests/Drupal/Tests/WebAssert.php:898
   ╵ /app/web/modules/contrib/localgov_directories/modules/localgov_directories_page/tests/src/Functional/PathIntegrationTest.php:94
   ╵ /app/vendor/phpunit/phpunit/src/Framework/TestResult.php:726
   ┴
```

### PR DESCRIPTION
**Pull request by stephen-cox**
fixes #198

### CODE REVIEW

**Review by finnlewis — APPROVED** [review]
Great - tests now passing, nice work @stephen-cox !

File: modules/localgov_directories_page/tests/src/Functional/PathIntegrationTest.php [hunk 1]
```diff
@@ -22,7 +22,7 @@ class PathIntegrationTest extends BrowserTestBase {
    *
    * @var string
    */
-  protected $profile = 'standard';
+  protected $profile = 'testing';
```
Comment (ekes) [inline modules/localgov_directories_page/tests/src/Functional/PathIntegrationTest.php:25]: To nit-pick can this not just get deleted? It's only needed if it's overriding the value.

**Review by ekes — APPROVED** [review]
For the rest placing the block makes sense :)

**Comment by finnlewis** [comment]
Leaving this one for you @stephen-cox

**Pull request merged**